main.go

```diff
@@ -121,50 +118,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	vcnClient, err := core.NewVirtualNetworkClientWithConfigurationProvider(ociAuthConfigProvider)
+	clientProvider, err := scope.NewClientProvider(region, authConfig)
```

Comment:
will we create clients for the home region here? if not we should and error out early